src/main/java/org/springframework/integration/dsl/SplitterEndpointSpec.java

```diff
@@ -34,13 +35,31 @@
 	}
 
 	/**
+	 * Set the applySequence flag to the specified value. Defaults to {@code true}.
 	 * @param applySequence the applySequence.
 	 * @return the endpoint spec.
 	 * @see AbstractMessageSplitter#setApplySequence(boolean)
 	 */
 	public SplitterEndpointSpec<S> applySequence(boolean applySequence) {
-		this.target.getT2().setApplySequence(applySequence);
+		this.handler.setApplySequence(applySequence);
 		return _this();
 	}
 
+	/**
+	 * Set delimiters to tokenize String values. The default is
+	 * <code>null</code> indicating that no tokenizing should occur.
+	 * If delimiters are provided, they will be applied to any String payload.
+	 * Only applied if provided {@code splitter} is instance of {@link DefaultMessageSplitter}.
+	 * @param delimiters The delimiters.
+	 * @return the endpoint spec.
+	 * @since 1.2
+	 * @see DefaultMessageSplitter#setDelimiters(String)
+	 */
+	public SplitterEndpointSpec<S> delimiters(String delimiters) {
+		if (this.handler instanceof DefaultMessageSplitter) {
```

Comment:
Not sure. It isn't critical option. I guess `warn` will meet your concern.
Something what we have for `requiresReply`:
```
if (this.handler instanceof AbstractReplyProducingMessageHandler) {
			((AbstractReplyProducingMessageHandler) this.handler).setRequiresReply(requiresReply);
		}
		else {
			logger.warn("'requiresReply' can be applied only for AbstractReplyProducingMessageHandler");
		}
```